app/components/work_packages/progress/status_based/modal_body_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= turbo_frame_tag "work_package_progress_modal" do %>
+  <%= primer_form_with(
+        model: work_package,
+        url: submit_path,
+        class: 'progress-form',
+        id: 'progress-form',
+        data: { 'application-target': "dynamic",
+                "work-packages--progress--preview-progress-target": "form",
+                controller: "work-packages--progress--focus-field work-packages--progress--preview-progress" }) do |f| %>
+    <%= flex_layout do |modal_body| %>
+      <% modal_body.with_row do |_fields| %>
+        <%= render(WorkPackages::ProgressForm.new(f, work_package:, mode:, focused_field:)) %>
+      <% end %>
+
+      <% if should_display_migration_warning? %>
+        <% modal_body.with_row(mt: 3) do |_migration_warning| %>
+          <%= render(Primer::Alpha::Banner.new) { t("work_package.progress.modal.migration_warning_text") } %>
+        <% end %>
+      <% end %>
+
+      <% modal_body.with_row(mt: 3) do |_tooltip| %>
+        <%= render(Primer::Beta::Text.new(font_weight: :semibold)) { t("work_package.progress.label_note") } %>
+        <%= render(Primer::Beta::Text.new) { t("work_package.progress.modal.status_based_help_text") } %>
+        <%= render(Primer::Beta::Link.new(href: "#")) { t(:label_learn_more) } %>
```

Comment:
Do we have a URL already for this?

Reply:
@aaron-contreras as suggested by Oliver, let's remove the links, and add them back once the documentation path is clear.

Reply:
@aaron-contreras as suggested by Parimal, it should link to https://www.openproject.org/docs/user-guide/time-and-costs/progress-tracking/.

Reply:
Done!